### DISCUSSION TITLE
Introduce Yourself To the Community -> Solidity Pathshala

### DISCUSSION THREAD

**Discussion by Vikash-8090-Yadav** [Vikash-8090-Yadav/Solidity-Pathshala#175] [General]
**Hello, :hand:  my name is Vikash Kumar Yadav and I'm from Varanasi . I am  blockchain / Web3 developr currently  exploring Devop's field , I built many project on the :spider_web:   Web3 , Blockchain , Ether .My teach stack is basically Docker :ship:  , Kub8 , React/Next for the Fronted  and Ether , Solidity ,  Hardhat for the Blockchain part.**  

If u have any doubt's Regarding any project Feel free to ask here in this discussion  group  only .  :ballot_box_with_check: 

 I'm excited to be here and hope to learn more about  Open source / Blockchain  and meet new people.
Let's Connect :world_map: 
All the social media link available  in my Github profile . 

Note : **All please introduce your self in this dsicussion**

**Comment by apoorva1823000** [comment]
Hello, ✋ Apoorva Mehta here from Gujarat . I am android developer with a keen interest in web 3 technologies , I built many project on the 🕸️ Web3 , Blockchain , as well as android using java .My teach stack is basically java, flutter for android development, python for software development as well as solidity for blockchain projects .

I'm excited to be here and hope to learn more about Open source / Blockchain and meet new people.
Let's Connect 🗺️
All the social media link available in my Github profile .

**Reply by Vikash-8090-Yadav** [reply]
NIce to meet you  @apoorva1823000  . Hope to see more contribution  from your side . :rocket:

**Comment by Syedhussainsaber** [comment]
Hey 🖐️, I am Syed Hussain Saber from Hyderabad, I am a frontend developer also now learning blockchain development. I hope that I can learn many more things by working as a contributor in this project 😊.

**Reply by Vikash-8090-Yadav** [reply]
Hey @Syedhussainsaber . Nice to see u in Solidity-Pathshala.  We both will rock together 🚀

**Comment by monishkajha17** [comment]
Hello my name is Monishka Jha from Delhi. I am a frontend developer and a Web 3 enthusiast. I had also work on many projects related to  web3,blockchain and IoT. In free time I love to mint NFTs. Hoping to learn something gr8 and to make good contributions.

**Reply by Vikash-8090-Yadav** [reply]
Hello @monishkajha17 . I am too NFT enthusiast . Glad to see u here . Looking forward to see gr8 contribution from your side . 

Let's rock together :rocket:

**Comment by suyashmadhesia** [comment]
Hey everybody I am Suyash Maddhessiya. I am final yeat student of BIET Jhansi. I have great interest in programming and can work on any technology. I am fast learner with eager and zill to learn new things. Let's connect on github.

**Reply by Vikash-8090-Yadav** [reply]
Great ! We ll rock together . Glad to see ur enthusiast . Let's  make  this  event more successful together ! :rocket:

**Comment by swarupyeole11** [comment]
Hello I am Swarup Yeole. I would love to connect and contribute.

**Reply by Vikash-8090-Yadav** [reply]
@swarupyeole11 excited to see contribution from your side . 😌💫

**Comment by Akash-Ghode** [comment]
Hello everyone, I am Akash from Mumbai. I am a Mechanical Engineer-2020 passout.
Past 1 year I am learning MERN stack and Blockchain. I have completed few unpaid CTFs and Ethernaut challenges.
Looking forward to develop my skills and contribute with Solidity Pathshala.

**Reply by Vikash-8090-Yadav** [reply]
Thanks for choosing Solidity-Pathshala.  
Mern stack + Blockchain =Boom 🚀 🔥💥.
looking forward to see ur contribution  in Solidity-Pathshala  . ❤️

**Comment by Saksham-1612** [comment]
Hii everyone, this is Saksham Srivastava. I know Python, Java , C++ , Solidity.
I have worked in MERN stack. Made one or two smart contracts with solidity also.
Well that's it. Lets build something crazy!!!

**Reply by Vikash-8090-Yadav** [reply]
Hey , Feels Gr8 to see u in Solidity-Pathshala.  Let's Rock together 🔥

**Comment by Sanitya7** [comment]
Hi all, 
I am Sanitya Srivastava, 
Building Recyteq [ www.recyteq.com ] , a startup, transforming human hair into plant based fertilizer. 
I like technology and blockchain is one of the future technologies, with the intent to make an impact I am in SWOC'23 to buidl solutions which can solve the present problems using blockchain. 

Currently, I have a basic understanding of C, Python and solidity. 
Looking ahead to try my hands out in advanced solidity concepts, truffle, hardhat and ganache-based projects. 

Let us also connect at Linkedin : 
https://www.linkedin.com/in/sanitya-srivastava/

**Reply by Vikash-8090-Yadav** [reply]
What startup idea :fire:  . Really impressed by your idea  . I already  know u and  u r so gr8 and pro enthusiast . 
Loved to see u again  here . 
Thanks for choosing solidity-Pathshala again . 
Let's rock together :rocket: :rocket:

**Comment by jovinkassi** [comment]
Hello , I am Jovin Johny Kassi. I am first year in Btech CSE. I know python and few concepts of javascript. I also watched alot of youtube videos regarding solidity and blockchain.I am keenly interested in this hackathon .Also i hope that i will receive support from you all,since this is my first time in any hackathon or related events. I would like to make my contribution to the project.


LinkedIn profile:
https://www.linkedin.com/in/jovin-johny-kassi-178131256/

**Reply by Vikash-8090-Yadav** [reply]
Gr8  .Looking forward to see contribution from your side . 💥

**Comment by prakhar3062** [comment]
I am Prakhar Mishra, a third-year BTech student. I know Python, C++, and Solidity.
I have worked in the MERN stack. I have worked on smart contracts with solidity using Remix Ide also.
Let us connect-:
https://www.linkedin.com/in/prakhar-mishra-39880520a/

**Reply by Vikash-8090-Yadav** [reply]
Looking forward  to see more contribution from your side . 
Thank's for choosing Solidity-Pathshala 

:rocket:

**Comment by SiddharthaMadhav** [comment]
Hi solidity community, I'm Siddhartha. I am a student currently pursuing my B.Tech. at SASTRA University in Tamil Nadu. I'm pretty new to the blockchain space and want to contribute to the open source and have fun.

**Reply by Vikash-8090-Yadav** [reply]
Looking forward  to see more contribution from your side . 
Thank's for choosing Solidity-Pathshala 

:rocket:

**Comment by Harshit-Raj-14** [comment]
Hello, ✋ my name is Harshit Raj and I'm from Patna. I am a second year student from VIT Bopal. I have worked in the MERN stack. I have worked on smart contracts with solidity using Remix Ide also.

So, I'm new to the blockchain space and want to contribute to the open source and have fun.

I'm excited to be here and hope to learn more about Open source / Blockchain and meet new people.
Let's Connect 🗺️ - https://www.linkedin.com/in/harshit-raj-500606229/

All the social media link available in my Github profile .

**Reply by Vikash-8090-Yadav** [reply]
Looking forward  to see more contribution from your side . 
Thank's for choosing Solidity-Pathshala 

:rocket:

**Comment by AnmolSirola** [comment]
Hello,  myself name Anmol Sirola and I'm from Delhi . I am blockchain / Web3 developer. I've built many project on the blockchain technology , I love participating in hackathons and I also like contributing to open source .
I'm excited to be here and hope to learn more about Open source / Blockchain and looking forward to meet and grow with you guys :)

**Reply by Vikash-8090-Yadav** [reply]
Looking forward  to see more contribution from your side . 
Thank's for choosing Solidity-Pathshala Again :heart:  

:rocket:

**Comment by TheNobody-12** [comment]
Hello Everyone My name is Sarthak Kapaliya. My Coding username is TheNobody-12. I am in 3rd-year of Computer Science at Pandit Deendayal Energy University, Gandhinagar.  I live in Ahmedabad, Gujarat.  My Github https://github.com/TheNobody-12 
I am new to solidity if anyone can tell me the good resources from which I can learn So I can contribute here? 

My Linkedin profile  https://www.linkedin.com/in/sarthak-kapaliya-2723bb202/

**Reply by Vikash-8090-Yadav** [reply]
Looking forward to see more contribution from your side .
Thank's for choosing Solidity-Pathshala Again heart

rocket

**Comment by Debopom-Banerjee** [comment]
Hello Myself Debopom Banerjee, CSE undergrad from West Bengal, India. I am a full stack blockchain developer and also a MERN stack Developer. I am an Open-source enthusiast and also learning DSA in recent times.

Let's connect : https://www.linkedin.com/in/debopom-banerjee-a35123215/

**Reply by Vikash-8090-Yadav** [reply]
Great to meet  u sir . Looking forward to see more contribution from your side . 
Thank's for choosing this repo. Soon i will add in the private channel of the Solidity pathshala and Web3 World :rocket:

**Reply by Vikash-8090-Yadav** [reply]
Blockchain+Mern stack === Boom :bomb: :fire:

**Comment by ankitBytes** [comment]
Hello everyone my name is Ankit Kumar Sahu  i am from odisha and a beginner to this field l. I am currently studying in 2nd year of my B. Tech course. I have knowledge on c, c++ and some knowledge on frontend development. Currently I am trying to complete DSA and want to explore the field of blockchain. 
I hope I would learn a lot new things in the by contributing in this project.

**Reply by Vikash-8090-Yadav** [reply]
Excited to work with u . @ankit2025 . Don't froget to star the repo as well as follw me :smile: 

Thank's for choosing Solidity-Pathshala

**Comment by vandit98** [comment]
hi everyone my name is vandit tyagi. I am a 2nd year B.Tech student at iiit naya raipur. I have knowledge of machine learning,python ,c++ and nosql. I want to learn blockchain and i feel like i am on the right place.

**Reply by Vikash-8090-Yadav** [reply]
Excited to see contribution from your side . Thanks for choosing Solidity-Pathshala . 
🚀✌️

**Comment by DevSwayam** [comment]
Hello everyone!

My name is swayam karle and I am excited to be a part of this community. I am currently a third year btech cs student. I am looking forward to learning from all of you and contributing to the discussions on this platform. Thank you for having me!

**Reply by Vikash-8090-Yadav** [reply]
Hlo @DevSwayam .Gr8 to see u here . Thanks for choosing Solidity-Pathshala for the contribution. Let's Rock together ✌️

**Comment by arpithams1947** [comment]
Hello, I am Arpitha M S and am from Bangalore. A former web developer and a web3 enthusiast. Currently learning about blockchain and its applications. Thank you for having me and I am excited to learn new things from this journey while contributing to the project. Happy to meet and connect with you all!

**Reply by Vikash-8090-Yadav** [reply]
Hey @arpithams1947 , Thank's for choosing  Solidity-Pathshala . Let me know if u have any doubt regarding Solidity-Pathshala . 

Looking forward to see contribution from your side . 
:smile:

**Comment by hrishik18** [comment]
Hello Everyone, 
My name is Hrishik Sancheti, a TY student in Computer Science (Engineering). I am excited to be a part of this community.
